tests/foreman/ui/test_computeresource_ec2.py

```diff
@@ -39,6 +39,7 @@ def module_ec2_settings():
     )
 
 
+@pytest.mark.e2e
 @pytest.mark.tier2
 @pytest.mark.skip_if_not_set('http_proxy')
 @pytest.mark.skip_if_open("BZ:2032530")
```

Comment:
Can we remove this `skip_if_open("BZ:2032530")`, as its already closed as wontfix since long

Reply:
We need to modify/update the test to make it work. I would remove this once we update the test.